packages/create-discord-bot/template/JavaScript/src/events/ready.js

```diff
@@ -1,5 +1,6 @@
 import { Events } from 'discord.js';
 
+/** @type {import('./index.js').Event<'ready'>} */
```

Comment:
```suggestion
/** @type {import('./index.js').Event<Events.ClientReady>} */
```

Reply:
> ```suggestion
> 
> /** @type {import('./index.js').Event<Events.ClientReady>} */
> 
> ```

Is there a specific reason you want to do this? It seems to be more of a stylistic choice.

Reply:
Because I think it's a good idea to use these enums when possible.